CMakeLists.txt

```diff
@@ -46,37 +48,44 @@ add_library(${PROJECT_NAME}_nodelet
 
 target_link_libraries(${PROJECT_NAME}_nodelet
   ${catkin_LIBRARIES}
+  ${OpenCV_LIBRARIES}
```

Comment:
Could also just add OpenCV_LIBRARIES to catkin_LIBRARIES so you don't have to do this for every thing :)